doc/codingGuidelines.md

```diff
@@ -148,16 +148,22 @@ In general, macros should be avoided (see [Modern C++](https://docs.google.com/d
 * Comments for users of classes and functions must be written in headers files. Comments in definition files are meant for contributors and maintainers.
 
 ### Include directive
+For source files (.cpp) with an associated header file (.h) that resides in the same directory, it should be `#include`'d with double quotes and no path.  This formatting should be followed regardless with whether the associated header is public or private. For example:
+```cpp
+// In foobar.cpp
+#include "foobar.h"
+```
+
 All included public header files from outside and inside the project should be `#include`’d using angle brackets. For example:
 ```cpp
 #include <pxr/base/tf/stringUtils.h>
 #include <mayaUsd/nodes/stageData.h>
 ```
 
-Private project’s header files should be `#include`'d using the file name when in the same folder. Private headers may live in sub-directories, but they should never be included using "._" or ".._" as part of a relative paths. For example:
+Private project’s header files should be `#include`'d using the quotes, and a relative path. Private headers may live in the same directory or sub-directories, but they should never be included using "._" or ".._" as part of a relative paths. For example:
```

Comment:
Towards the beginning: "using double quotes" instead of "using the quotes"
Towards the end: "as part of a relative path." instead of "as part of a relative path**s**."(plural)

Reply:
Fixed in 2a53f95a0fae1e22002e158ba7985dacf9a9b3c7